server/cluster.go

```diff
@@ -139,25 +139,32 @@ func (s *Server) getRaftCluster() (*raftCluster, error) {
 		return s.cluster, nil
 	}
 
-	// Find in etcd
+	return nil, nil
+}
+
+func (s *Server) createRaftCluster() error {
+	if s.cluster.IsRunning() {
+		return nil
+	}
+
 	value, err := getValue(s.client, s.getClusterRootPath())
 	if err != nil {
-		return nil, errors.Trace(err)
+		return errors.Trace(err)
 	}
 	if value == nil {
-		return nil, nil
+		return nil
```

Comment:
Seems not, if we have not bootstrapped the cluster, then the cluster value from etcd will be nil.
@siddontang  Am i right?